cleverhans/utils_tf.py

```diff
@@ -64,20 +40,18 @@ def initialize_uninitialized_global_variables(sess):
         sess.run(tf.variables_initializer(not_initialized_vars))
 
 
-def model_train(sess, x, y, predictions, X_train, Y_train, save=False,
-                predictions_adv=None, init_all=True, evaluate=None,
-                feed=None, args=None, rng=None, var_list=None):
+def model_train(sess, loss, x, y, X_train, Y_train, save=False, init_all=True,
```

Comment:
This change needs to be backward compatible. Could you add 6 months deprecation warnings: e.g., saying that loss will have to be provided in the future etc...

Reply:
Yes. Basically I ended renaming the new function to model_train_loss.

Reply:
I'd vote to just call the new function `train`. And we can leave `model_train` in place for 6 months for backwards compatibility.

Reply:
Done